### ISSUE TITLE
Unlike Narrator, NVDA does not read the notification for no available app updates in the Microsoft Store

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#7984]
### Steps to reproduce:
1. Open Microsoft Store.
2. TAB to the "See more" button and activate it, to show a menu with additional options/commands.
3. From the menu, with the up/down arrow keys, select the "Downloads and Updates" item and activate it with ENTER.
4. TAB to the "Get Updates" button and activate it with ENTER.
5. Wait for the check for updates to complete. If there are updates available, wait for them to complete updating and repeat step 4. If not, go to step 6.
6. Observe what will NVDA report when there are no updates available. Note, that if there are no updates available, Microsoft Store will display a notification about it above the list of recently updated apps.
7. Repeat the steps from 1 to 6 again, this time using Narrator.
8. Observe what Narrator will report when there are no updates available. Note, that if there are no updates available, Microsoft Store will display a notification about it above the list of recently updated apps.

### Expected behavior:
NVDA should report the displayed on the screen message about no available updates, as Narrator does.

### Actual behavior:
NVDA does not report the notification about no available updates.

### System configuration:
- Operating system: Windows 10 Pro version 1709 (build 16299.214), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: next-14846,f24c74c2.
- Microsoft Store version: 11801.1001.6.0.
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: Palit GeForce GT 630, 1024MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

**Comment by josephsl** [comment]
Hi,

Confirmed.

Technical: either name change event isn't picking up the correct object (in which case I can fix it by probing for things and editing my add-on), or notification UIA event must be caught (requires a fix from NVDA Core). The UIA notification event is part of build 16299 and later.

CC @MichaelDCurran

**Comment by josephsl** [comment]
Hi,

Update: #8009 provides a more general case.

Thanks.

**Comment by Brian1Gaff** [comment]
You know my feelings about what microsoft seem to be doing lately. they have 
never heard the proverb, if its not broken don't fix it!
 grin.

**Comment by josephsl** [comment]
Hi, in this case, they did publish some materials about it, so both sides are equally at fault (I think). Thanks.

**Comment by Adriani90** [comment]
This works as expected in NVDA 2019.1.1. Closing as works for me.
